R/ael01_nollt.R

```diff
@@ -72,7 +82,11 @@ ael01_nollt_pre <- function(adam_db,
 #' @returns the postprocessing function returns an `rlistings` object or an `ElementaryTable` (null report).
 #'
 ael01_nollt_post <- function(tlg, ...) {
-  if (nrow(tlg) == 0) tlg <- null_report
+  if (is(tlg, "rl_list")) {
```

Comment:
this is quite common post processing right? and we may think of removing empty listing component in that list

Reply:
Good point but the `as_listing` function is already doing it. I added a test.